kratos/utilities/geometry_tester.cpp

```diff
@@ -944,8 +961,74 @@ bool GeometryTesterUtility::VerfiyShapeFunctionsValues(
 /***********************************************************************************/
 /***********************************************************************************/
 
+bool GeometryTesterUtility::VerifyShapeFunctionsSecondDerivativesValues(
+    GeometryType& rGeometry,
+    GeometryType::CoordinatesArrayType& rGlobalCoordinates,
```

Comment:
I know all other test have the same signature, but you don't need mutable references here, right? It'd put my mind at ease if references were immutable unless you wanted to manipulate the objects they point to.

Reply:
I prefer to maintain the same signature for all tests. There is no point in changing this one if all are implemented in the same way.